packages/app-builder/src/routes/ressources+/settings+/analytics+/filters.ts

```diff
@@ -0,0 +1,45 @@
+import { type ExportedFields } from '@app-builder/models/data-model';
+import { isAdmin } from '@app-builder/models/user';
+import { initServerServices } from '@app-builder/services/init.server';
+import { getRoute } from '@app-builder/utils/routes';
+import { type ActionFunctionArgs, redirect } from '@remix-run/node';
+
+export async function action({ request }: ActionFunctionArgs) {
+  const { authService } = initServerServices(request);
+  const { user, dataModelRepository } = await authService.isAuthenticated(request, {
+    failureRedirect: getRoute('/sign-in'),
+  });
+
+  if (!isAdmin(user)) {
+    return redirect(getRoute('/'));
+  }
+
+  // * GET * //
+
+  if (request.method === 'GET') {
+    try {
+      const dataModel = await dataModelRepository.getDataModel();
+
+      const exportedEntries = await Promise.all(
+        dataModel.map(async (table) => {
+          const exported = await dataModelRepository.getDataModelTableExportedFields(table.id);
+          return [table.id, exported] as const;
+        }),
+      );
+      const exportedFieldsByTable = Object.fromEntries(exportedEntries) as Record<
+        string,
+        ExportedFields
+      >;
+
+      return Response.json({ success: true, exportedFieldsByTable });
+    } catch (err) {
+      console.error(err);
+      return Response.json(
+        { success: false, error: 'Failed to get exported fields' },
+        { status: 500 },
+      );
+    }
+  }
```

Comment:
Be careful, this is an action, Remix should not be happy by calling a mutation with the GET method.
Furthermore I don't think we ever cared about HTTP methods in actions, this is just adding complexity